plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -392,8 +434,7 @@ public function enhance_product_collection_with_interactivity( $block_content, $
 				$block_content = $this->enable_client_side_navigation( $block_content );
 			}
 		}
-
-		return $block_content;
+		return sprintf( '<div>%s</div>', $block_content );
```

Comment:
We add this `div` to center the product list on the front end when we select a percentage (%) `width`.